apache/incubator-metron
# METRON-867: In the event that we graduate, remove incubating from the website and documentation

## Contributor Comments
NOTE:
* This is not an indication of graduation status.
* I did not change any link to the archives of our lists on the website.

To test this:
* Spin up full-dev and make sure things still work (I touched one python file, but I think it's fine).
* Spin up the website from this PR and make sure no incubator logos or other indication of incubation.
* Generate the docs and ensure they contain no indication of incubation

## Pull Request Checklist

Thank you for submitting a contribution to Apache Metron (Incubating).  
Please refer to our [Development Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61332235) for the complete guide to follow for contributions.  
Please refer also to our [Build Verification Guidelines](https://cwiki.apache.org/confluence/display/METRON/Verifying+Builds?show-miniview) for complete smoke testing guides.  


In order to streamline the review of the contribution we ask you follow these guidelines and ask you to double check the following:

### For all changes:


### For documentation related changes:

  ```
  cd site-book
  bin/generate-md.sh
  mvn site:site
  ```

#### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.
It is also recommended that [travis-ci](https://travis-ci.org) is set up for your personal repository such that your branches are built there before submitting a pull request.
